src/doc/trpl/crates-and-modules.md

```diff
@@ -265,11 +239,9 @@ another crate.
 We have a library crate. Let's make an executable crate that imports and uses
 our library.
 
-Make a `src/main.rs` and put this in it: (it won't quite compile yet)
+Make a `src/main.rs` and put this in it:
```

Comment:
yeah you'd think that but when we don't mention it upfront, people come into IRC because they think they've done something wrong :( Some people don't consider that it may not compile until you actually tell them.